packages/modules/templates/ApplicationDetails/components/PropertyDocuments.js

```diff
@@ -25,7 +25,7 @@ function PropertyDocuments({ documents }) {
     <div style={{ marginTop: "19px" }}>
       {documents?.map((document, index) => (
         <React.Fragment key={index}>
-          <CardSubHeader style={{ marginBottom: "8px", color: "#505A5F", fontSize: "24px" }}>{t(document?.title)}</CardSubHeader>
+          <CardSubHeader style={{ marginTop:"32px",marginBottom: "8px", color: "#505A5F", fontSize: "24px" }}>{t(document?.title)}</CardSubHeader>
```

Comment:
Did you check all instances of this component and if used verify alignment didn't break